content/js/users.md

```diff
@@ -140,6 +140,11 @@ While each of the sign-up functions is resolved with a user object,
 in most cases you need not deal with it because
 you can access the currently logged-in user using `skygear.currentUser`.
 
+If your app does not require the user to create an account before using your
+app, you can sign up the user anonymously. An user that is signed up
+anonymously does not have a username, email, nor a password. You can add a
+username, email and/or password to a user after signing up.
```

Comment:
Not sure why people keep making distinction between users and users that are signed up anonymously. They are the same.

The function was added as a convenient function to create a user without username, email nor password. It essentially calls the underlying implementation that is also called by the `signup` function.

Additionally, the function was added to bring feature parity to the iOS SDKs which also have this method for a long time.

I thought this is pretty clear what it means to sign up anonymously: “An [sic] user that is signed up anonymously does not have a username, email, nor a password.”